.github/workflows/publish-chart.yml

```
@@ -3,25 +3,11 @@ name: Publish OCI Helm Chart
-        id: extract
-        run: |
-          TAG_REF="${GITHUB_REF#refs/tags/}"
-          VERSION_WITHOUT_V="${TAG_REF#v}"
-          echo "version=$VERSION_WITHOUT_V" >> "$GITHUB_OUTPUT"
   publish:
     runs-on: ubuntu-latest
-    needs:
-      - version
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)